pkg/plugins/policies/meshtls/plugin/v1alpha1/plugin_test.go

```diff
@@ -217,11 +228,7 @@ var _ = Describe("MeshTLS", func() {
 														},
 													}},
 													Default: meshhttproute_api.RuleConf{
-														BackendRefs: &[]common_api.BackendRef{{
-															TargetRef: builders.TargetRefMeshService("backend", "", "test-port"),
```

Comment:
a backendRef can't have a sectionName and it needs to _not_ have a port to refer to a kuma.io/service

Reply:
@lukidzi does the test still make sense as is? Probably it should have a Dataplane with inbound `kuma.io/service: backend` to really be complete but since we don't check endpoints here it seems to be ok without